templates/error.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
     <title>Scribble.rs - Error</title>
-    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"/>
```

Comment:
I might not be that familiar with html, not sure why did you have to add a viewport metadata?

Reply:
It is testing.  I should not delete <meta charset="UTF-8" />.
the viewport is done for media query CSS for all devices.